### PR TITLE
feat: [SRM-13213]: Account level SLO integration

### PR DESCRIPTION
**Pull request by deepesh-ui**
#### Screenshots

Screenshot 2022-12-08 at 10 33 01 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by deepesh-ui** [comment]
retrigger jest

**Review by tan-nhu — APPROVED** [review]

File: src/modules/85-cv/pages/slos/CVSLODetailsPage/CVSLODetailsPage.tsx [hunk 1]
```diff
@@ -43,8 +43,10 @@ const CVSLODetailsPage: React.FC = () => {
 
   const projectIdentifierRef = useRef<string>()
   const isCompositeSLO = sloType === SLOType.COMPOSITE
+  const isAccountLevel = !orgIdentifier && !projectIdentifier && !!accountId
+  const pathQueryParams = isAccountLevel ? { accountId } : { accountId, orgIdentifier, projectIdentifier }
   useEffect(() => {
-    if (projectIdentifierRef.current && projectIdentifierRef.current !== projectIdentifier) {
+    if (!isAccountLevel && projectIdentifierRef.current && projectIdentifierRef.current !== projectIdentifier) {
       history.push(routes.toCVSLOs({ accountId, orgIdentifier, projectIdentifier }))
       return
```
Comment (praneshg239) [inline src/modules/85-cv/pages/slos/CVSLODetailsPage/CVSLODetailsPage.tsx:51]: no need of this return here

File: src/modules/85-cv/pages/slos/CVSLOsListingPage.tsx [hunk 2]
```diff
@@ -253,12 +276,18 @@ const CVSLOsListingPage: React.FC<CVSLOsListingPageProps> = ({ monitoredService
...
     const slo = row?.original
     const { name = '', sloIdentifier = '', description = '', sloType = '' } = slo || {}
-    const path = routes.toCVSLODetailsPage({
-      identifier: sloIdentifier,
-      accountId,
-      orgIdentifier,
-      projectIdentifier
-    })
+    const path =
+      orgIdentifier && projectIdentifier
```
Comment (praneshg239) [inline src/modules/85-cv/pages/slos/CVSLOsListingPage.tsx:280]: use `isAccountLevel` flag here for better understanding

File: src/modules/85-cv/pages/slos/components/CVCreateSLOV2/CVCreateSLOV2.tsx [hunk 3]
```diff
@@ -194,7 +196,7 @@ const CVCreateSLOV2 = ({ isComposite }: { isComposite?: boolean }): JSX.Element
 
   const links = [
     {
-      url: routes.toCVSLOs({ accountId, orgIdentifier, projectIdentifier, module: 'cv' }),
```
Comment (praneshg239) [inline src/modules/85-cv/pages/slos/components/CVCreateSLOV2/CVCreateSLOV2.tsx:197]: Add conditional redirect here

File: src/modules/85-cv/pages/slos/components/CVCreateSLOV2/components/CreateCompositeSloForm/components/AddSlos/components/SLOList.utils.tsx [hunk 4]
```diff
@@ -22,10 +22,19 @@ export const getUpdatedSLOObjectives = (
   const weight = Number(100 / selectedSlosLength).toFixed(1)
   const lastWeight = Number(100 - Number(weight) * (selectedSlosLength - 1)).toFixed(1)
   const updatedSLOObjective = selectedSlos.map((item, index) => {
+    const orgAndProjectIdentifiers =
+      orgIdentifier && projectIdentifier
```
Comment (praneshg239) [inline src/modules/85-cv/pages/slos/components/CVCreateSLOV2/components/CreateCompositeSloForm/components/AddSlos/components/SLOList.utils.tsx:26]: Update to `isAccountLevel`

**Review by praneshg239 — APPROVED** [review]

**Pull request merged**